site/en/workflows.md

```diff
@@ -551,6 +551,7 @@ processors with calls to `ocrd-tesserocr-recognize` with specific parameters:
 
 * `segmentation_level` determines the *highest level* to segment. Use `"none"` to disable segmentation altogether, i.e. only recognize existing segments.
 * `textequiv_level` determines the *lowest level* to segment. Use `"none"` to segment until the lowest level (`"glyph"`) and disable recognition altogether, only analyse layout.
+Be aware that if you use this parameter with `ocrd-tesserocr-recognize`, it will resegment the words anyway.
```

Comment:
```suggestion
Be aware that if you use `-P segmentation_level word -P textequiv_level word` with `ocrd-tesserocr-recognize`, it will resegment the words anyway.
```

Is that what you mean?

Reply:
yes, thanks for clarifying